internal/cmd/marketplace/apply/apply.go

```diff
@@ -49,6 +48,26 @@ miactl marketplace apply -f ./path/to/myFantasticGoTemplate.json -f ./path/to/my
 
 # Apply all the valid configuration files in the directory myFantasticGoTemplates to the Marketplace
 miactl marketplace apply -f myFantasticGoTemplates`
+
+	// applyEndpoint has to be `Sprintf`ed with the companyID
+	applyEndpoint = "/api/backend/marketplace/tenants/%s/resources"
+
+	imageKey    = "image"
+	imageURLKey = "imageUrl"
+
+	supportedByImageKey    = "supportedByImage"
+	supportedByImageURLKey = "supportedByImageUrl"
+)
+
+var (
+	errCompanyIDNotDefined = errors.New("companyID must be defined")
+
+	errResWithoutName       = errors.New(`the required field "name" was not found in the resource`)
+	errNoValidFilesProvided = errors.New("no valid files were provided, see errors above")
```

Comment:
Idk, including 'see errors above' within the text of another error seems to me a little too hardwired.

Reply:
```suggestion
	errNoValidFilesProvided = errors.New("no valid files were provided")
```
True, removing it